### PR TITLE
[AC-7728] P1: Office hours add to calendar does not honor timezones

### PR DESCRIPTION
**Pull request by geofrocker**
### Changes introduced in: [AC-7728](https://masschallenge.atlassian.net/browse/AC-7728) :
- Added timezones to the payload
### How to test on test2
- Override a mentor email with your email
- Schedule office hours on behalf of that mentor
- Reserve office hours on behalf of a finalist
- Notice that the email you receive has office hours links and an .ics file that honors timezones
### N.B Waiting for permission to raise add2cal PR

### CODE REVIEW

File: web/impact/requirements/base.txt [hunk 1]
```diff
@@ -53,4 +53,4 @@ django-oidc-provider==0.7.0
 django-sitetree==1.13.1
 django-graphql-jwt==0.1.10
 
-git+https://github.com/masschallenge/add2cal
+git+https://github.com/masschallenge/add2cal#AC-7728
```
Comment (jonkiparsky) [inline web/impact/requirements/base.txt]: Presumably this branch tag will need to be removed after the AC-7728 branch on add2cal is merged, correct?

**Review by jonkiparsky — CHANGES_REQUESTED** [review]

File: web/impact/requirements/base.txt (hunk 1 above)
Comment (geofrocker) [reply web/impact/requirements/base.txt]: Correct

**Review by shanbady — APPROVED** [review]
validated functionality here including ics attachment. Will need to remember to switch the branch on-merge.otherwise looks good. approving.

**Review by freshprincekla — APPROVED** [review]
QA looks good.
- I was able to schedule an office hour
- reserve the office hours and received an email with a .ics file that honors the timezones

**Review by jonkiparsky — APPROVED** [review]

**Pull request merged**